src/code42cli/options.py

```diff
@@ -79,13 +79,44 @@ def set_debug(ctx, param, value):
 pass_state = click.make_pass_decorator(CLIState, ensure=True)
 
 
+profile_option_quiet = click.option(
+    "--profile",
+    expose_value=False,
+    callback=set_profile,
+    hidden=True,
+    help="The name of the Code42 CLI profile to use when executing this command.",
+)
+debug_option_quiet = click.option(
```

Comment:
Instead of having to duplicate these definitions, I think we could make each of these options and `sdk_options` take a boolean as a parameter to indicate whether or not they are hidden. The default could be true since that's what we would want most of the time. Something like:

```python
def profile_option(hidden=True):
    return click.option(
        "--profile",
        expose_value=False,
        callback=set_profile,
        hidden=is_hidden,
        help="The name of the Code42 CLI profile to use when executing this command.",
    )

def sdk_options(hidden=True):
    f = profile_option(hidden)
    ...
    return f

@sdk_options(hidden=False)
def some_command()
    pass
```

Reply:
I like it, and went a step further and make `@sdk_options` take the argument instead of two separate decorators.

Reply:
💥